woocommerce/woocommerce-ios
# [Free Trial] Add Upgrades Menu Button UI

Closes: #9056  

# Why

This PR adds an "Upgrades" item in the Hub Menu. The item is only visible for WPCom sites.

# Screenshot

Self Hosted Site | WPCom site
--- | ---
<img width="434" alt="hosted-site" src="https://user-images.githubusercontent.com/562080/225758353-723ad53c-b7a2-4e98-9675-29cf2cbd88ac.png"> | <img width="442" alt="wpcom-site" src="https://user-images.githubusercontent.com/562080/225758356-f3e847d4-4521-4513-b711-e24eaefa1891.png">

# Testing

- Log into a self-hosted site
- Go to the hub menu
- See no "Upgrades" item.

----

- Log into a self-hosted site
- Go to the hub menu
- See the new "Upgrades" item.


---
